index.html

```diff
@@ -193,6 +193,14 @@
 The resulting profiles are then used for many purposes
 including advertising, credit ratings, and market analysis.
 
+Some credential systems demand live interactions with an issuer
+to validate presentations
+or use linkable presentations.
```

Comment:
This could be read as "Some credential systems demand live interactions with an issuer
to ... use linkable presentations.", which I don't think you mean. Reversing it avoids that reading:
```suggestion
Some credential systems use linkable presentations or
demand live interactions with an issuer
to validate presentations.
```